### PR TITLE
[15.0][MIG] helpdesk_mgmt_rating: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Locked by:
- [x] `helpdesk_mgmt` https://github.com/OCA/helpdesk/pull/391

@Tecnativa TT37283

### CODE REVIEW

File: helpdesk_mgmt_rating/data/helpdesk_data.xml [hunk 1]
```diff
@@ -11,85 +11,118 @@
...
-                          </td>
-                      </tr>
-                  </table>
-              </td></tr>
-              <tr><td style="padding:15px 20px 10px 20px;">${(object.user_id.signature or '')| safe}</td></tr>
-          </tbody>
-      </table>
-      <table style="width:600px;margin:auto;text-align:center;font-size:12px;">
+        <field name="body_html" type="html">
+<div>
```
Comment (pedrobaeza) [inline helpdesk_mgmt_rating/data/helpdesk_data.xml]: Why changing the indentation?

**Review by pedrobaeza — COMMENTED** [review]
Please change the Spanish translation. Due to the high work done on the module (unit tests, mail templates, migration), we deserve to be co-authors since this version.

**Comment by victoralmau** [comment]
Changes done.

**Comment by victoralmau** [comment]
Now, it's ready to review again.

**Review by pedrobaeza — APPROVED** [review]
You have to clean translations of the mail template as well in the migration script. Thanks for adding the migration script though.

**Comment by victoralmau** [comment]
Ping @chienandalu

**Review by chienandalu — APPROVED** [review]
Functional and code review :+1:

**Pull request merged**